server/src/sim.rs

```diff
@@ -30,6 +30,7 @@ pub struct Sim {
     graph: DualGraph,
     spawns: Vec<Entity>,
     despawns: Vec<EntityId>,
+    graph_entities: GraphEntities,
```

Comment:
Oh, interesting! Previously, only the client kept track of this. No wonder it was so well-hidden.